### ISSUE TITLE
Keyboard navigation control does not work for calendar.

### ISSUE THREAD

**Issue by jzwoo** [jzwoo/pe#1]
Issue description: Keyboard navigation control for the navigation of calendar does not seem to be working on my Mac, while command box focus seems to be working as expected. 

Expected result: Switching between calenders.

Actual results: Calendar does not switch. 

However, I do acknowledge that there is a command line option for the navigation of calendars which is a good fallback should the keyboard navigation fails.
